src/kubernetes/server/server-config.yaml

```diff
@@ -37,8 +37,19 @@ data:
               kubernetes:
                 accounts:
                   default:
+                    imagePullSecret: harborsec
                     limits:
                       memory: 1024Mi
+          container:
+            metadata:
+              disableSslVerification: true
+              registry-configurations:
+                - registry-host: 'myregistrydomain.com:5000'
+                  authorization-type: basicauth
+                  user: testuser
+                  secret: testpassword
+                - registry-host: registry-1.docker.io
+                  authorization-type: dockerhub
```

Comment:
Another naming question, why call this `dockerhub` - as there are so many registries out there giving it a product specific name doesn't seem correct.

Reply:
I've renamed `dockerhub` to `dockerouaht2`